.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 npm-debug.log
 docker-modem/
+.project
```

Comment:
I can't speak for @apocas, but usually these sort of editor choice specific entries go in one's personal `~/.gitignore` instead of every project they work on.